### PR TITLE
[1.19.2] Full ru_ru wiki localization

### PR DESCRIPTION
**Pull request by CrossVas**
This PR contains localization material for IC2Classic's Wiki manual for ru_ru language. 
The order of wiki entries follows the order they appear in game wiki manual (for easier maintenance). 
There might be some small very hard to detect spelling mistakes, but overall, these are very rare, as I reformatted every entry to fit the page and provide adequate comprehensive information about every block IC2Classic add.

Note 1: This is a very massive amount of work (for one person), so there might be mistakes. If you find a mistake or some piece of information that would require some more details, feel free to contact me and I'll be glad to provide support.  

Note 2: I'm not a native speaker of given language, I studied it at the university, but I do know it at a decent level, so again, I'm ready to fix any mistakes, if found.

### CODE REVIEW

**Comment by Speiger** [comment]
Yeah don't worry.
I wrote the majority on my own so i know how much work it is :)
I'll leave it open until the next patch gets started :)

Anything specific i should wait for?

**Comment by CrossVas** [comment]
Yeah, just let me do a quick check over other translation. Will be able to finish until you start working on new update (is it's not tomorrow :"D)

**Comment by Speiger** [comment]
Eh the update won't start until next weekend :)
So there is a lot of time.

**Comment by CrossVas** [comment]
Seems pretty solid right now.

File: src/main/resources/assets/ic2/lang/wiki_ru_ru.json
Comment (TheYxel) [inline src/main/resources/assets/ic2/lang/wiki_ru_ru.json:1]: @CrossVas 
I found some typos.
"wiki.ic2.subpage.brewing.body":
...
Чтобы собрать бочку, вам нужно её **стракфтить** **в** поместить на Деревянные Леса.

Чтобы собрать бочку, вам нужно её скрафтить и поместить на Деревянные Леса.

File: src/main/resources/assets/ic2/lang/wiki_ru_ru.json [hunk 1]
```diff
@@ -1,699 +1,683 @@
...
... удар от кого либо труднее (обычно наоборот).\nТ.е. вас уносит настолько, что тактика \"пьяного мастера\" кажется отличной защитой.§<newPage>§<header;Приготовление Зелий;true>\nПриготовление зелий тоже очень простое.\n\nКак и обычная варочная стойка, но в промышленных масштабах.\nПросто поставьте ингредиенты в большом количестве и всё готово.\n\nК примеру, для Зелья Огнестойкости вам нужно:\nНезерский нарост, Сгусток Магмы и вода.\nВода начнёт процесс.\n\nТакже можете заранее кинуть до 20 шт. красного камня или светокаменной пыли.\n- §lКрасный камень§r: Увеличивает время (до 30 минут)\n\n\n- §lСветокаменная пыль§r: Увеличивает уровень (до 3 уровня).\n\nНо стоит учесть, что каждый доп. уровень уменьшает продолжительность вдвое.\n\nКачество зависит от продолжительности и уровня.§<newPage>§<header;Получение зелья;true>\nЛучшее качество достигается примерно за 4 часа.\n\nМы можете разом получить до 12 бутылочек или 120 стрел с зельем.\n\nЭтого хватит, пока варится новая партия.§<newPage>§<header;Переварка зелий;true>\nПосле 20 часов варки залье портится и превращается в ядовитое облако рядом с бочкой при открытии, так что не надо оставлять без присмотра.\n\nЭто касается только зелий!",
```
Comment (TheYxel) [inline src/main/resources/assets/ic2/lang/wiki_ru_ru.json:110]: @CrossVas I found some typos.
Чтобы собрать бочку, вам нужно её **стракфтить** **в** поместить на Деревянные Леса.
Чтобы собрать бочку, вам нужно её скрафтить и поместить на Деревянные Леса.

Comment (CrossVas) [reply src/main/resources/assets/ic2/lang/wiki_ru_ru.json:110]: @TheYxel fixed, thank you. Any other mistakes?

Comment (TheYxel) [reply src/main/resources/assets/ic2/lang/wiki_ru_ru.json]: @CrossVas have not noticed any more yet. You did a titanic job )

**Comment by TheYxel** [comment]
@CrossVas looks like it still have some more typos...
[ruruwiki_typos.txt](https://github.com/TinyModularThings/IC2Classic/files/11324553/ruruwiki_typos.txt)

**Comment by CrossVas** [comment]
> @CrossVas looks like it still have some more typos... [ruruwiki_typos.txt](https://github.com/TinyModularThings/IC2Classic/files/11324553/ruruwiki_typos.txt)

@TheYxel 
Damn, feel so bad. Fixed all of them, except the word "не легко", as it is an antonym to the word "легко" and should be written together as "нелегко".

**Comment by CrossVas** [comment]
@TheYxel The description for The Thermometer was too long, so I changed it a bit. Thanks!

image

image

File: src/main/resources/assets/ic2/lang/wiki_ru_ru.json [hunk 2]
```diff
@@ -1,699 +1,683 @@
...
+  "wiki.ic2.subpage.second_tier.body": "Вы достигли точки, где захотите продвинуться на следующий уровень.\nЧто для этого нужно?\n\n- Источник энергии, который выдаёт как минимум 50-80EU/т.\n\n- Куча грязи, глины или терракоты.§<newPage>§<header;Алюминий;true>\nДля продвинутых механизмов, вам нужен алюминий.\n\nЕго можно получить в §<redirect;true;Редкоземельном Экстракторе;ic2:preview.rare_earth_extractor> с помощью грязи, глины, терракоты. \n- Получение из §lГрязи§r самый медленный способ, но самый лёгкий\n- С §lГлиной, Терракотой§r будет быстрее, но их сложнее добыть.§<newPage>§<header;Алюминий в Незере;true>\nАлюминий также можно добыть в Незере, но он очень хорошо спрятан. \nОднако 1-2 блока руды могут обеспечить вас алюминием надолго. \n\nКак только вы получили алюминиевую пыль, её можно переплавить в §<redirect;true;Заводе Сплавов;ic2:preview.alloy_smelter> ипользуя песок.\n\n\nВот где вам понадобится электричество, Завод Сплавов очень прожорливый механизм. \nУбедитесь, что ваша сеть выдержит его аппатит и не опустеет!§<newPage>§<header;Список Механизмов;true>",
```
Comment (elias506) [inline src/main/resources/assets/ic2/lang/wiki_ru_ru.json:100]: "аппатит" must be "аппетит"

File: src/main/resources/assets/ic2/lang/wiki_ru_ru.json [hunk 3]
```diff
@@ -1,699 +1,683 @@
...
...<newPage>§<header;Направление энергии;true>\nПрактическое правило:\n- Одна точка - §lВыход§r\n- 3 точки или их отсутствие - §lВход§r.\nНебольшое примечание: цвета точек показывают, какая мощность будет на выходе / входе, в зависимости от соответствующих по цвету кабелей.§<newPage>§<header;Разделение сети;true>\nТеперь есть еще одна проблема, с которой вы можете или не можете столкнуться.\nЧто произойдет, если вам нужно проложить кабели очень близко друг к другу и вы не хотите, чтобы они соединялись?\n\nУ вас есть 2 решения этой проблемы:\n- Сдаться. \n- Сделать сеть шире§<newPage>§<header;Валики;true >\nШутки в сторону. Вариант 1: §<redirect;true;Валики с краской;ic2:preview.painter> §<image;ic2:textures/wiki/color_blocking.png;45>\n§<padding;5>\nОкрашенные кабели не подключаются к другим кабелям, если они разного цвета.§<newPage>§<header;Шахтёрские трубы;true>\nВаш второй вариант — шахтёрские трубы: §<image;ic2:textures/wiki/mining_blocking.png;45>\n§<padding ;5>\nШахтёрские трубы могут быть размещены на кабель и действовать как анкеры / заглушки, и кабели не будут соединяться друг с другом.\n\nКроме того, держа их в руке, сносный графический индикатор подскажет куда их ставить.",
```
Comment (elias506) [inline src/main/resources/assets/ic2/lang/wiki_ru_ru.json:105]: "По этому при работе с машинами стоит проверить" in this case must be spelled in "Поэтому"

**Comment by elias506** [comment]
There are lots of lexical errors guys. If necessary, I can fix it

**Comment by elias506** [comment]
And what will start the problem when the text does not fit on the page completely? Maybe you can somehow add a page opening, as done in other examples

**Comment by CrossVas** [comment]
@elias506 
> There are lots of lexical errors guys. If necessary, I can help to fix it

Would be appreciated.

> And what will start the problem when the text does not fit on the page completely? Maybe you can somehow add a page opening, as done in other examples

Currently adding **§< newPage >§< header;< TITLE >;true >** to any **_preview_** pages will prevent localization from loading when in game, or even crash the game on first load.

**Comment by elias506** [comment]
> Currently adding **§< newPage >§< header;< TITLE >;true >** to any **_preview_** pages will prevent localization from loading when in game, or even crash the game on first load.

That sucks because it doesn't make sense to translate many pages. I think this is an important bug.

**Comment by Speiger** [comment]
> > Currently adding **§< newPage >§< header;< TITLE >;true >** to any **_preview_** pages will prevent localization from loading when in game, or even crash the game on first load.
> 
> That sucks because it doesn't make sense to translate lots of pages. I think this is an important bug.

This isn't really a bug this is design choice.
The preview pages are designed to be short summaries of the Elements.

**Pull request merged**